src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GeneralType.php

```diff
@@ -99,6 +99,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'label' => $this->trans('Recalculate shipping costs after editing the order', 'Admin.Shopparameters.Feature'),
                 'help' => $this->trans('Automatically updates the shipping costs when you edit an order.', 'Admin.Shopparameters.Help'),
+            ])
+            ->add('autoremove_not_valid_cartrules', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Autoremove cart rules not valid from cart', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Automatically removes expired or no valid cart rules from cart.', 'Admin.Shopparameters.Help'),
```

Comment:
I think you can remove this help text. The label speaks for itself. :-)

See the [Content style guide](https://zeroheight.com/80a6c5a61/v/latest/p/510e8a-components/b/574c7e)